Packs/ANYRUN/Integrations/AnyRunSandbox/AnyRunSandbox.yml

```diff
@@ -0,0 +1,1476 @@
+          name: env_locale
+          defaultValue: 'en-US'
+          required: false
+          type: String
+        - description: 'Network connection state.'
+          name: opt_network_connect
+          defaultValue: 'True'
+          required: false
+          type: String
+        - description: 'FakeNet feature status.'
+          name: opt_network_fakenet
+          defaultValue: 'False'
+          required: false
+          type: String
+        - description: 'TOR using.'
+          name: opt_network_tor
+          defaultValue: 'False'
+          required: false
+          type: String
+        - description: 'Tor geo location option. Example: US, AU.'
+          name: opt_network_geo
+          defaultValue: 'fastest'
+          required: false
+          type: String
+        - description: 'HTTPS MITM proxy option.'
+          name: opt_network_mitm
+          defaultValue: 'False'
+          required: false
+          type: String
+        - description: 'Residential proxy using.'
+          name: opt_network_residential_proxy
+          defaultValue: 'False'
+          required: false
+          type: String
+        - description: 'Residential proxy geo location option. Example: US, AU.'
+          name: opt_network_residential_proxy_geo
+          defaultValue: 'fastest'
+          required: false
+          type: String
+        - auto: PREDEFINED
+          description: 'Privacy settings.'
+          name: opt_privacy_type
+          defaultValue: 'bylink'
+          required: false
+          type: String
+          predefined:
+            - 'public'
+            - 'bylink'
+            - 'owner'
+            - 'byteam'
+        - description: 'Timeout option. Size range: 10-660.'
+          name: opt_timeout
+          defaultValue: '240'
+          required: false
+          type: String
+        - auto: PREDEFINED
+          description: 'Start file analysis from the specified directory.'
+          name: obj_ext_startfolder
+          defaultValue: 'temp'
+          required: false
+          type: String
+          predefined:
+            - 'desktop'
+            - 'home'
+            - 'downloads'
+            - 'appdata'
+            - 'temp'
+            - 'windows'
+            - 'root'
+        - description: 'Optional command line.'
+          name: obj_ext_cmd
+          required: false
+          type: String
+        - description: 'Forces the file to execute with elevated privileges and an elevated token
+            (for PE32, PE32+, PE64 files only).'
+          name: obj_force_elevation
+          defaultValue: 'False'
+          required: false
+          type: String
+        - description: 'Change extension to valid.'
+          name: obj_ext_extension
+          defaultValue: 'True'
+          required: false
+          type: String
+      name: anyrun-detonate-file-windows
+      description: 'Perform File analysis using Windows VM.'
+      outputs:
+        - contextPath: ANYRUN.TaskID
```

Comment:
Using a generic and identical key in the context of several different commands may make it difficult for a user to distinguish between the results of different runs of different commands.
it should be some unique context path for each command